commands2/__init__.py

```diff
@@ -42,6 +42,7 @@
 from .sequentialcommandgroup import SequentialCommandGroup
 from .startendcommand import StartEndCommand
 from .subsystem import Subsystem
+from .sysid import SysIdRoutine
```

Comment:
We shouldn't re-export things from a subpackage at the top level. The `sysid` package won't be useful for the majority of robot programs, so the typical robot program shouldn't pay the cost of importing all sysid logging code.

Reply:
Handled